app/controllers/sign_up/passwords_controller.rb

```diff
@@ -45,10 +45,13 @@ def permitted_params
 
     def process_successful_password_creation
       password = permitted_params[:password]
+      now = Time.zone.now
       UpdateUser.new(
         user: @user,
-        attributes: { password: password, confirmed_at: Time.zone.now },
+        attributes: { password: password, confirmed_at: now },
       ).call
+      @user.email_addresses.update(confirmed_at: now)
```

Comment:
Same concern here about confirming all email addresses, though i think at this point the user should only have 1 email address.